### ISSUE TITLE
TODO - mandag

### ISSUE THREAD

**Issue by Hansstem** [Hansstem/TDT4173-ML#5]
**Ole Jacob**: 
- [x] Linear regression
- [x] random forest
- [x] Gradient boosting

**Hans**: 
- [x] Support vector machine
- [x] Time series forecasting: LST and GRU
- [x] Neural Networks
- [x] LightGBM

**Peter**: 
- [x] Time series models, ARIMA og Prophet
- [x] Anomaly detection
- [x] Ensemble models
